src/style/Banner.css

```diff
@@ -0,0 +1,8 @@
+.banner {
+  position: fixed;
+  text-align: center;
+  background: #005191;
```

Comment:
Used colors from the existing UI. If we have new hex codes someone could update these.

Reply:
Nice, thanks!